### PR TITLE
Add AddAuthorizeAttribute

### PR DESCRIPTION
**Pull request by zouql**
Add AddAuthorizeAttribute Method

### CODE REVIEW

**Comment by liangshiw** [comment]
Have you tested it ? Is it applicable to 3. x and 2. x project

**Comment by zouql** [comment]
y

**Comment by zouql** [comment]
This is very difficult?

**Comment by zouql** [comment]
It is applicable to 3. x and 2. x project

**Comment by liangshiw** [comment]
Thanks , I whill check this

File: src/LogDashboard/LogDashboard.csproj [hunk 1]
```diff
@@ -51,16 +49,21 @@
     <None Remove="css\Simple-Line-Icons.woff2" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.1'">
+    <PackageReference Include="Microsoft.AspNetCore.Authorization" Version="3.0.0" />
```
Comment (liangshiw) [inline src/LogDashboard/LogDashboard.csproj]: 你可以将这两个包都设定到3.0.0版本吗,这样可以兼容3.0 - 3.x 的项目

File: src/LogDashboard/LogDashboard.csproj [hunk 2]
```diff
@@ -51,16 +49,21 @@
...
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="3.0.1" />
+  </ItemGroup>
+
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
-   
+    <PackageReference Include="Microsoft.AspNetCore.Authorization" Version="2.2.0" />
```
Comment (liangshiw) [inline src/LogDashboard/LogDashboard.csproj]: 同样,请将版本设定到2.0.0版本

File: src/LogDashboard/LogDashboard.csproj [hunk 3]
```diff
@@ -51,16 +49,21 @@
...
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.2.0" />
   </ItemGroup>
+  
   <ItemGroup>
-
     <PackageReference Include="Dapper" Version="2.0.30" />
     <PackageReference Include="DapperExtensions.Standard" Version="1.1.0" />
-    <PackageReference Include="Microsoft.AspNetCore.Http.Abstractions" Version="2.0.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Authorization.Policy" Version="2.2.0" />
```
Comment (liangshiw) [inline src/LogDashboard/LogDashboard.csproj]: 这里也是

**Comment by liangshiw** [comment]
Thanks a lot

**Pull request merged**